CMakeLists.txt

```diff
@@ -12,6 +12,9 @@ option(ENABLE_PROGRAMS "Build mbed TLS programs." ON)
 
 option(UNSAFE_BUILD "Allow unsafe builds. These builds ARE NOT SECURE." OFF)
 
+# export the submodule flag so that crypto knows it's being built as a submodule
+set( USE_CRYPTO_SUBMODULE ON )
```

Comment:
Have a look at https://github.com/ARMmbed/mbedtls/pull/2349/files#diff-af3b638bc2a3e6c650974192a53c7291R12

Reply:
I opted for removing it entirely, since it's not an option anymore (and Mbed TLS doesn't check for it anywhere). Should I changed that?

Reply:
No need to change. "It's not an option anymore" is a good reason.